docs/configuration.md

```diff
@@ -1664,6 +1664,16 @@ Apart from these, the following properties are also available, and may be useful
         </td>
     </tr>
     <tr>
+        <td><code>spark.ssl.port</code></td>
+        <td>0</td>
+        <td>
+            Port number to listen on for SSL connections.
+            The SSL port should be between 1024 and 65535 (inclusive). 
+            Default value of 0 means the port will be determined automatically.
+            Attention that the port should be separated for each particular protocols.
```

Comment:
I don't think it's clear what the last sentence means. I'd either remove it or say something like "the port can be specified for services individually, with properties like `spark.ssl.YYYY.port`, as described above."